contents/docs/integrate/_snippets/snippet.mdx

```diff
@@ -0,0 +1,34 @@
+import Snippet from "../snippet.mdx"
+
+### Quick start: JavaScript snippet (recommended)
+
+> This is the simplest way to get PostHog up and running on your website, and only takes a few minutes to set-up. (This method generally works best for non-technical users or for very simple websites which don't need advanced tracking.) If you're using npm, or want to instrument custom events directly from your code, visit the dedicated [JavaScript library](/docs/integrate/client/js) page.
```

Comment:
```suggestion
> This is the simplest way to get PostHog up and running on your website, and only takes a few minutes to set-up.
```
In general I think the snippet is actually better, even for technical users or advanced tracking. It's often times harder to reason about `posthog-js` and the semantics around when it gets loaded, and the snippet allows you to enqueue events immediately as the page loads. You also still get access to all the same features since the snippet effectively just loads `posthog-js`, so in general I think we should encourage people to use it whenever possible, unless a framework-specific wrapper exists.